internal/utils/parse_cli.go

```diff
@@ -0,0 +1,56 @@
+package utils
+
+import (
+	"strings"
+)
+
+func ParseCliArguments(arguments []string) ([]string, map[string]interface{}) {
```

Comment:
There should be another return parameter to show if there is an error.

    func ParseCliArguments(arguments []string) ([]string, map[string]interface{}, err Error) {

Reply:
Addressed.